libraries/AP_ESC_Telem/AP_ESC_Telem.h

```diff
@@ -91,11 +94,17 @@ class AP_ESC_Telem {
     // udpate at 10Hz to log telemetry
     void update();
 
+    // is ESC telemetry working
+    bool is_active() const { return _have_data; }
```

Comment:
I don't mind too much but from a casual observer, it's not clear to me what the difference is between, "is_active" and "is_telemetry_active".  Does one refer to the motors being active and the other to the telemetry being active?  Is it just that one allows providing a mask?  In which case the names should perhaps be the same but one of them gets a different suffix.  So for example "is_active" and "is_active_by_mask".. or "is_telemetry_active_all" and "is_telemetry_active".  I'm not sure.. just throwing ideas out there to try and make it more clear.

Reply:
Removed